CASMSMF-7509: Fresh install customization.yaml changes to autorize PCIM GUI

shreni123 opened the pull request:
## Summary and Scope

_Summarize what has changed. Explain why this PR is necessary. What is impacted? Is this a new feature, critical bug fix, etc?_

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves - https://jira-pro.it.hpe.com:8443/browse/CASMSMF-7509

## Testing

_List the environments in which these changes were tested._

### Tested on:

Tyr
Hela

![image](https://github.com/Cray-HPE/csm/assets/53111642/76c94c84-8a61-454d-954b-139cac9d29aa)

![image](https://github.com/Cray-HPE/csm/assets/53111642/3454e822-0010-4493-bc06-fc4f3b19ff19)

djlapenta commented:
John Heemstra mentioned in the ticket @shreni123:
-------------
for CSM versions greater than 1.4, this change needs to be made via a single PR in the shasta-cfg repo: https://github.com/Cray-HPE/shasta-cfg

Once that PR merges, we'll "git vendor" it into both the release/1.5 and release/1.6 branches in the csm repo.
--------------

heemstra committed:
Update "shasta-cfg" from "git@github.com:Cray-HPE/shasta-cfg.git@main"

git-vendor-name: shasta-cfg
git-vendor-dir: vendor/github.com/Cray-HPE/shasta-cfg
git-vendor-repository: git@github.com:Cray-HPE/shasta-cfg.git
git-vendor-ref: main

heemstra committed:
Squashed 'vendor/github.com/Cray-HPE/shasta-cfg/' changes from a4f24d83..51d59da2

51d59da2 Merge pull request #11 from Cray-HPE/CASMMON-352
0940cb20 CASMMON-352: Fresh install customization.yaml changes to autorize PCIM CDU GUI in shasta environment
5cd99510 Merge pull request #10 from Cray-HPE/CASMTRIAGE-5750
460db5ef CASMTRIAGE-5750: fix spire compute issue
05e78f03 Merge pull request #9 from Cray-HPE/CASMHMS-6072
3502a453 CASMHMS-6072: Add customizations to the cray-hms-discovery chart to create the default river default credentail sealed secret.

git-subtree-dir: vendor/github.com/Cray-HPE/shasta-cfg
git-subtree-split: 51d59da2dba8be42df597bb2427f7be723f72548
